### PR TITLE
Fix #5565 - Handle missing col in usage in 0.10.4

### LINKED ISSUE #5565: Handle missing column in usage fqn match
fix for 0.10.4 and 0.11

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by harshach — APPROVED** [review]

File: ingestion/src/metadata/ingestion/sink/metadata_rest.py [hunk 1]
```diff
@@ -279,7 +279,6 @@ def write_topics(self, topic: Topic) -> None:
         try:
             topic_request = CreateTopicRequest(
                 name=topic.name,
-                displayName=topic.displayName,
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/sink/metadata_rest.py:282]: Checked the JSON Schema on 0.10.4 and it does not have displayName. Might be from another patch
